code/maxPerf.vcxproj

```diff
@@ -22,7 +22,7 @@
     <VCProjectVersion>15.0</VCProjectVersion>
     <ProjectGuid>{D0B15FD1-8D51-4033-B19F-477FAAF59787}</ProjectGuid>
     <RootNamespace>maxPerf</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>
```

Comment:
We could switch `WindowsTargetPlatformVersion`, but we might want to check with everyone before pushing this change to the `master` branch.